### DISCUSSION TITLE
Two input devices - keyboard and mouse

### DISCUSSION THREAD

**Discussion by nemanjam** [sezanzeb/input-remapper#961] [Q&A] [answered]
I want to map extra left taster on mouse + `shift` as input to produce `Control_L + v` output. I am unable to select or record two inputs. How do I do that?

**Comment by jonasBoss** [comment, marked as answer]
Unfortunately, this is currently a limitation in the design of InputRemapper. It is not easily possible to use the inputs of two devices in the same mapping. There might be a way to do this with multiple mappings and macros.

To solve this, we would need to finish #592 and rework the GUI as well as the concept of a preset. So I don't think this will happen any time soon.
